pkg/authorino/authorino.go

```diff
@@ -17,7 +17,7 @@ func Authorino(ns string) *authorinov1beta1.Authorino {
 			Namespace: ns,
 		},
 		Spec: authorinov1beta1.AuthorinoSpec{
-			Image:       "quay.io/3scale/authorino:v0.6.0",
+			Image:       "quay.io/3scale/authorino:v0.7.0",
```

Comment:
This can be omitted now that the Authorino Operator [defaults to a fixed version](https://github.com/Kuadrant/authorino-operator/blob/c9c6f90930594125b774e73c6d67cbc4f5dafd02/api/v1beta1/authorino_types.go#L53) of Authorino.

No harm to keep it though if you want.

Reply:
I knew you were going suggest that :) 

I did it just to be explicit. But yes, manifests are pinned and the refer to tagged images, so there is no risk breaking it.